_includes/version-with-upgrade-notice.html

```diff
@@ -0,0 +1,10 @@
+<div class="alert alert-warning news-container-alert mt-4">
+  <a class="news-overlay" href="https://yarnpkg.com/getting-started/migration#why-should-you-migrate"></a>
+  <div class="news-inner">
+    These instructions only cover Yarn versions prior to 2.0. They entered <a href="https://en.wikipedia.org/wiki/Maintenance_mode">maintenance mode</a> in January 2020 and will eventually reach their end-of-life in terms of support. To see the up-to-date documentation, check the main website: <a href="https://yarnpkg.com/getting-started">yarnpkg.com/getting-started</a>.
+    <br>
+    The latest Yarn version is: <img alt="Latest CLI Release" src="https://img.shields.io/npm/v/@yarnpkg/cli/latest?label=latest">
```

Comment:
I think we should point to the migration guide rather than getting-started, since people accessing this install page will likely already know what Yarn is, and will be more interested by either the install instructions or migration guide.

```suggestion
    <div>
      These instructions only cover Yarn versions prior to 2.0. Those versions entered <a href="https://en.wikipedia.org/wiki/Maintenance_mode">maintenance mode</a> in January 2020 and will eventually reach their end-of-life in terms of support. Please see the main website for the most up-to-date documentation: <a href="https://yarnpkg.com/getting-started/migration#why-should-you-migrate">yarnpkg.com/getting-started/migration</a>.
    </div>
    <div style="margin-top: 15px">
      The latest Yarn version is: <img alt="Latest CLI Release" src="https://img.shields.io/npm/v/@yarnpkg/cli/latest?label=latest">
    </div>
```

Reply:
I was initially thinking about putting the migration guide, but then in the notice you wrote for me to put on the website, you put the getting-started page and after some thought, I managed to convince myself that it's fine. :P 

Since the entire notice already has an overlay to `https://yarnpkg.com/getting-started/migration#why-should-you-migrate`, I think we should rather link the v2+ installation page instead.

What do you think?

Reply:
It's likely most people won't see that the overlay is clickable - having an explicit link is still a good idea, imo.

As for the target, people on the Classic website will likely be in a state where they want to work on a Classic project - by which point knowing how to install Yarn 2 will matter less than knowing how to migrate it altogether 🤔